lib/net.js

```diff
@@ -133,6 +134,7 @@ function Socket(options) {
   this._handle = null;
   this._parent = null;
   this._host = null;
+  this._timeout = null;
```

Comment:
Again, not that I'm able to see: https://github.com/nodejs/node/blob/master/lib/timers.js#L296-L317